vas2nets/tasks.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import requests
+
+from django.conf import settings
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url).content.decode('utf-8')
+
+        return list(csv.reader(content.splitlines(), delimiter=','))[1:]
+
+    def run(self, date, **kwargs):
+        data = self.get_data(date)
+        for row in data:
+            VoiceCall.objects.create(
+                created_at=row[0],
+                msisdn=row[2],
```

Comment:
What about the shortcode field that they give us? We can store that fairly easily, and we might want it for future queries, especially if we want to compare stats between the 55500 and the 1444 line.